packages/components/src/Tag/Tag.stories.tsx

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+import * as React from "react";
+import Tag, { stylesByColor } from "./Tag";
+import type { Color } from "./Tag";
+import type { Story } from "@storybook/react";
+import WarningIcon from "../SVGIcon/Icons/Warning";
+import styles from "./Tag.stories.module.css";
+
+const colorOptions: Array<Color> = Object.keys(stylesByColor);
+
+export default {
+  title: "Tag",
+  component: Tag,
+
+  argTypes: {
+    color: {
+      control: {
+        type: "select",
+        options: colorOptions,
+      },
+    },
+  },
+};
+
+type Args = React.ElementProps<typeof Tag>;
```

Comment:
`React.ElementProps` doesn't exist, and this should be `React.ComponentProps`.

We need to be careful when moving Flow stuff over that we convert it, since apparently we don't have TypeScript being checked on CI 😬

Reply:
We do have TS checking in the [build step](https://github.com/chanzuckerberg/edu-design-system/runs/3146904330), I think this one is from story files not being checked